manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -618,24 +630,24 @@ func (r *FybrikApplicationReconciler) checkGovernanceActions(configEvaluatorInpu
 			ProcessingLocation: geo,
 		}
 
-		actions, err := LookupPolicyDecisions(req.Context.DataSetID, resMetadata, r.PolicyManager, appContext, &reqAction)
+		actions, _, err := LookupPolicyDecisions(req.Context.DataSetID, resMetadata, r.PolicyManager, appContext, &reqAction)
```

Comment:
why do you ignore message here and in the case of errors, here and below, drop previously return message ?

Reply:
These calls are done to evaluate the potential storage accounts that can be used in case of copy, what actions need to be taken should the copy be done. We refer only to the message for "real" requests not including copy.

Reply:
Again, can you add comments, pls

Reply:
Added comments